asab/metrics/influxdb.py

```diff
@@ -196,11 +196,33 @@ def metric_to_influxdb(metric_record, now):
 			# SKIP empty fields
 			if not field.get("values") or field.get("values") == {}:
 				continue
-			values_lines.append(build_metric_line(field.get("tags"), field.get("values")))
+			values_lines.append(build_metric_line(field.get("tags"), (field.get("values"))))
 
 	return ["{},{} {}\n".format(name, line, int(timestamp * 1e9)) for line in values_lines]
 
 
+def validate_name(name: str):
+	return name.replace(" ", "\\ ").replace(",", "\\,")
+
+
+def sanitize_tags(tags: dict):
+	sanitized: dict = {}
```

Comment:
```python
sanitized = dict()
```

Reply:
"sanitized" is terrible word :laughing:

Reply:
i wasn't the one who came up with it, but i didn't know any other good word for it

Reply:
fixed, changed to cleanup instead